daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/VersionTimeline.scala

```diff
@@ -14,6 +14,10 @@ import scalaz.{-\/, @@, NonEmptyList, OneAnd, Ordering, Semigroup, \&/, \/, \/-}
 
 import scala.language.higherKinds
 
+// FIXME: https://github.com/digital-asset/daml/issues/7788
+// VersionTimeline is currently being rewritten and simplified.
+// Comments may be inaccurate.
```

Comment:
@martin Do you think this comment is fine ?

Reply:
Yes, I think that's ok.